api/apps/articles/model/save.coffee

```diff
@@ -62,11 +62,13 @@ getDescription = (article) =>
   text
 
 removeStopWords = (title) ->
-  title = title.replace(/[.,\/#!$%\^&\*;:{}=\—_`’~()]/g," ")
-  newTitle = _.difference(title.split(' '), stopWords.stopWords).join(' ')
-  return newTitle
+  title = title.replace(/[0-9.\/#!$%\^\*;{}=_`’~()]/g,"")
```

Comment:
I think we should keep numbers as non stop words. I think for lists, sometimes the number can be memorable and I could imagine users potentially using it in a search (thinking '30 under 30' and similar). I think that we could prob keep units as none stop words as well. Everything else looks good to me!